test/cpp/interop/xds_interop_client.cc

```diff
@@ -154,11 +156,16 @@ class TestClient {
             std::chrono::system_clock::now() + std::chrono::seconds(INT_MAX);
       }
     }
+    SimpleRequest request;
+    request.set_response_size(absl::GetFlag(FLAGS_response_payload_size));
+    std::string payload(absl::GetFlag(FLAGS_request_payload_size), '\0');
+    request.mutable_payload()->set_body(
```

Comment:
Since `Payload payload = 3;` is a message (therefore it's nullable, we should only set the `payload` when `request_payload_size` > 0 to avoid sending unnecessary data.

Reply:
Fixed